Sources/SwiftSDKGenerator/Extensions/String+hasPrefixIn.swift

```diff
@@ -0,0 +1,10 @@
+extension String {
+    func hasPrefix(in array: [String]) -> Bool {
```

Comment:
This seems a bit more readable with this slightly more verbose name
```suggestion
    func hasAnyPrefix(from array: some String]) -> Bool {
```